site/docs/v1/tech/tutorials/prometheus.adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: doc
+title: Prometheus Setup
+description: Learn how to setup Prometheus to work with FusionAuth
+---
+
+== Prometheus Setup
+
+[NOTE.since]
+====
+Available since `1.26.0`
+====
+
+Prometheus is a tool to monitor metrics and setup alerts in the case that some metric has responded negatively for an extended period. Setting this up with FusionAuth is easy.
+
+=== Setup
+The only requirement to hookup FusionAuth is to direct prometheus to the FusionAuth prometheus endpoints. We used the following configuration for a FusionAuth instance running locally on the docker host. (You can reach dockers host ip via `host.docker.internal` on mac and windows docker.)
+
+[source,yaml]
+.prometheus.yaml
+----
+include::docs/src/yaml/prometheus.yaml[]
+----
+
+We then started prometheus with
+[source,bash]
+----
+docker run -v $(pwd)/prometheus.yaml:/etc/prometheus/prometheus.yml -p 9090:9090 prom/prometheus
+----
+
+image::tutorials/setting-up-prometheus/prometheus-graph.png[Prometheus Graph View,width=1200,role=shadowed]
```

Comment:
Please add a 'configuration' section.

You could point to https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config and say that this is a typical `scrape_config`. 

Specific configuration questions I had that we should answer:
* Is the username always `apikey`? 
* I am guessing but am unsure that the `password` is the API key?
* What permissions does the API key need to have?

Reply:
Do you not see the yaml file? That is the configuration. Or do you want me to rename "setup" section to configuration?

Reply:
I do see the YAML file, there are just things I don't understand in it :), like the questions I asked. So I think it's worth having a configuration section like this (or something similar; I don't understand the basic auth stuff, so feel free to change anything below that is incorrect):

## Configuration

The Prometheus endpoint is configured with the YAML file, but there are some FusionAuth specific twists.

* The endpoint is protected with basic auth, but using the string `apikey` as the username and a valid API key as the password.
* The API key used for authentication must have such and such permissions at a minimum.

All other configuration is similar to a standard `scrape_config`. You can learn more about the [standard options here](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).